### ISSUE TITLE
pmx_plot_individual error: object 'isobserv' not found

### ISSUE THREAD

**Issue by tmss1** [ggPMXdevelopment/ggPMX#235]
**Describe the bug**
`pmx_plot_individual` fails with the error: `Error in FUN(X[[i]], ...) : object 'isobserv' not found`

**Screenshots**
![image](https://user-images.githubusercontent.com/32617414/169795724-9c057cd8-361d-4115-9525-f80d6c403aeb.png)

**Additional context**
The issue only happens with this specific project dataset but I cannot understand why. 
The issue is specific to `pmx_plot_individual`, other plots seem to work fine. 
The same model using a previous version of this dataset containing less data with only the PK endpoint worked fine. 
The new dataset, containing more endpoints, was filtered to include only the PK endpoint (as in the previous dataset) but with some additional data points. 
The issue persists whether the dataset was filtered inside or outside Monolix.

**Comment by mattfidler** [comment]
Is there any way to add a minimally reproducible example here?

**Comment by mattfidler** [comment]
Hi @tmss1 

I am unclear why this is happening, but by reading the code `isobserv` is a calculated variable.  I have added a fallback of `isobserv` to `TRUE` in the dataset.  

While it would be better to get a small reproducible example, perhaps this default value would work in your situation.

You can try by:

```r
remotes::install_github("ggPMXdevelopment/ggPMX", ref="235-pmx_plot_individual-error-object-isobserv-not-found")
```

Depending on your environment you may have to create a local library and then use:

```r
new_lib <- tempfile()
dir.create(new_lib)
with_libpaths(new_lib, {
remotes::install_github("ggPMXdevelopment/ggPMX", ref="235-pmx_plot_individual-error-object-isobserv-not-found")
library(ggPMX)
}, action="prepend")
``` 
Then you can see if it works for this project

**Comment by mattfidler** [comment]
Fixed in e8a540f but don't have a test ☹️
